standalone-node/installation_scripts/bootable-usb-prepare.sh

```diff
@@ -267,12 +270,31 @@ copy_files() {
     os_filename=$(printf "%s\n" usb_files/*.raw.gz 2>/dev/null | head -n 1)
```

Comment:
- No check/creation for /mnt directory before mounting. Ensure /mnt exists and is not in use before mounting.
- If rsync is not installed, attempts to fallback to cp, but both could be more robust (e.g., what if both fail?). 
- The function does not always unmount /mnt if it fails before the umount call.Always unmount /mnt using a trap.